_posts/2018-11-14-sr2019-rules-published.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: news
+title: SR2019 Rules Published & Useful Resources
+---
+
+We had a successful Kickstart on Saturday and it was great to see the excited teams getting familiar with our kit.
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the other robots within the zone boundaries. ([full rules][rules])
```

Comment:
Since we're going to mention it in the social media posts too, perhaps include a mention of the multiple levels of the volcano? (@antoinepetty your framing in slack was good, maybe adapt this towards that?)